src/utils/rtl.tsx

```diff
@@ -180,3 +181,111 @@ function getTextFromTableCellNode(node: ChildNode) {
 export const withBeamRTL: Wrapper = {
   wrap: (c) => <BeamProvider>{c}</BeamProvider>,
 };
+
+/** Selects an option from the Beam SelectField, MultiSelectField, and TreeSelectField components
+ * For select fields that support multiple selections, subsequent calls to this function will toggle the selection state of an option.
+ * @param value The value or label of the option.
+ * */
+export function select(element: HTMLElement, value: string | string[]) {
+  const select = resolveIfNeeded(element);
+  if (isSelectElement(select)) {
+    throw new Error("Beam select helper does not support <select> elements.");
+  }
+
+  if (!isInputElement(select)) {
+    throw new Error(`Expected element to be INPUT, but got ${select.nodeName}. This field may be read-only.`);
+  }
+
+  function maybeOpenAndSelect(optionValue: string) {
+    const listboxId = select.getAttribute("aria-controls");
+    const body = select.closest("body") as HTMLElement;
+    const listbox = body.querySelector(`#${listboxId}`) as HTMLElement;
+    if (!listbox) {
+      _click(select);
+    }
+    const options: NodeListOf<HTMLElement> = body.querySelectorAll("[role=option]");
```

Comment:
Question: Can we scope this to nodes within the `listbox`?

Reply:
Yes, we can. Makes sense to do. I will update.